package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "start": "node ./bin/www",
     "dev": "DEBUG=starter-code:* nodemon ./bin/www",
-    "dev-windows": "nodemon ./bin/www"
+    "dev-windows": "nodemon ./bin/www",
+    "seeds": "nodemon .bin/seed"
```

Comment:
I think "node" will do instead of "nodemon". When executing this file, the application is not started, so there is no need to have the reload function of nodemon applied here.